docs/administration-guide/operation-score.md

```diff
@@ -0,0 +1,186 @@
+# Operation Scores
+
+Operation Scores can be used to give an OperationChains and NamedOperations a "score" which can then be used to determine whether a particular user has the required permissions to execute a given OperationChain.
+
+These scores work on a credit-like system where user's configured "score" is the amount they can then spend on running an Operation/OperationChain. 
+Users cannot spend more than their score and every time they run an Operation or OperationChain that has a configured "score" this will be 'subtracted' from their score 'credit'.
+
+## Using Operation Scores
+
+A `ScoreOperationChain` operation determines a "score" for an OperationChain.
+To use this Operation you need to configure an `OperationsDeclarations.json`.
+
+To configure this file you need to set the following handlers which configure how Operation Scores are used with a graph:
+
+- `opScores` - required map of operation scores. These are the operation score values.
+- `authScores` - required map of operation authorisation scores. These are the maximum scores allowed for a user with a given role. 
+- `scoreResolver` - some operations may require a custom way of calculating an associated score, therefore an implementation of the [`ScoreResolver`](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/store/operation/resolver/ScoreResolver.html) interface may be required. 
+These map the operation class to its respective score resolver.
+There is a `DefaultScoreResolver` to which the custom implementation should delegate, in a manner specific to the new Operation.
```

Comment:
This duplicates the content in the "Score Resolver" section to some extent. I think it could be moved into there and this description could then be brief with a link to the next section for more info.